lib/msal-common/src/cache/entities/CredentialEntity.ts

```diff
@@ -108,17 +108,17 @@ export class CredentialEntity {
      * @param key
      */
     static getCredentialType(key: string): string {
-        // First keyword search will match all "AccessToken" and "AccessToken_With_AuthScheme" credentials
-        if (key.indexOf(CredentialType.ACCESS_TOKEN.toLowerCase()) !== -1) {
-            // Perform second search to differentiate between "AccessToken" and "AccessToken_With_AuthScheme" credential types
-            if (key.indexOf(CredentialType.ACCESS_TOKEN_WITH_AUTH_SCHEME.toLowerCase()) !== -1) {
-                return CredentialType.ACCESS_TOKEN_WITH_AUTH_SCHEME;
+        const separator = Separators.CACHE_KEY_SEPARATOR;
+        const domainRe = "(https?:\\/\\/)?([\\w-]+\\.)*([\\w-]{1,63})(\\.(\\w{2,3}))(\\:[0-9]{4,5})?";
+
+        for (const credType of Object.entries(CredentialType)) {
```

Comment:
I don't think Object.entries is supported in IE11/Legacy Edge which we, unfortunately, still need to support until msal-browser v3 to avoid breaking changes. Can we use Object.keys instead?

Reply:
Good point. Replaced with `Object.keys()`.